### DISCUSSION TITLE
userChromeJS Manager button broken in Firefox 110.0

### DISCUSSION THREAD

**Discussion by nollinvoyd** [xiaoxiaoflood/firefox-scripts#243] [General Support]
I posted in "issues".  The button is still on the menubar, but it doesn't work. Still works in Fx 109.01.

I tried looking thru all the documentation fro the custom buttons.  Is there an update for this, or a fix?

**Comment by xiaoxiaoflood** [comment]
Already fixed a month ago. userChromeJS and scripts don't have autoupdate, so you need to manually download the latest version of files and replace them in your profile folder. In this case, *rebuild_userChrome.uc.js*.

**Comment by nollinvoyd** [comment]
Thanks, all good now.
